proto/shared_constants.proto

```diff
@@ -140,8 +140,8 @@ message PortType {
 /*
  * Port behaviors
  */
-message PortBehavior {
-  enum Behavior {
+message DeviceEvent {
```

Comment:
PortBehavior was better, these (as a message construct) aren't "events" -- they may be used as an event, but that's a use of it, not what it is.  PortBehavior, SequestedState, SequesteredBehavior all seem appropriate to me.

DeviceBehavior wouldn't be good because it's not actually the behavior of the device.

PortState is too generic (port up/down?), so also not good here.

Reply:
Got it, changed back to PortBehavior. PTAL..